tests/materialize/materialize-mysql/snapshot.json

```diff
@@ -701,7 +701,7 @@
 {
   "row": {
     "date": "0000-01-01",
-    "datetime": "0000-00-00 00:00:00",
+    "datetime": "0000-01-01 00:00:00",
```

Comment:
That's a good find, I don't see a way out of this for us. Am I right that postgres is also limited to microsecond precision? Per the table [here](https://www.postgresql.org/docs/current/datatype-datetime.html):

> resolution 1 microsecond

Reply:
When evaluating loads, i'd expect `2023-08-30T14:13:22.000664521Z` to be auto-truncated to the `fsp` of the datetime type. Same thing when doing stores. So it kinda seems like it would just work?

You could _technically_ get different materialized views for different selections of `fsp`, but so long as the load an store evaluations are consistent, the result should be correct for the given `fsp`, no?